cards/src/main/resources/cards/custom/group2/set_633/spell_enveloping_mists.json

```diff
@@ -1,21 +0,0 @@
-{
```

Comment:
Needs the call to `removeCards(...)` in the migration if this card is getting deleted. consider just setting it to `"collectible": false` instead.